upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.8.yaml.template

```diff
@@ -98,12 +98,15 @@ spec:
         # for cluster communication
         - effect: NoSchedule
           operator: Exists
+      # Minimize downtime during a rolling upgrade or deletion; tell Kubernetes to do a "force
+      # deletion": https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods.
+      terminationGracePeriodSeconds: 0
```

Comment:
Hmm, have you tested this behavior?  Does calico handle being killed like this gracefully, I imagine there's some cleanup on shutdown...

Reply:
All the manifests (if not all, we missed updating them) on the Calico docs site have been updated to include `terminationGracePeriodSeconds: 0` so this being set has seen a decent amount of testing.
There is no cleanup I'm aware of on calico-node shutdown.